app/controllers/invitation_controller.rb

```diff
@@ -0,0 +1,20 @@
+class InvitationController < ApplicationController
+
+  def new
+
+  end
+
+  def create
+    github_service = GithubService.new(current_user)
+    user = github_service.get_user(params[:handle])
+    github_user = GithubUser.new(user)
+
+    if github_user.email && current_user.github_token
```

Comment:
Could you even get a github_user with an email back if the current_user doesn't have a token? Is this redundant?

Reply:
No, I don't think you could, you're right. I wanted to protect against the possibility of a user putting in a post request somehow, even if they are not connected to Github. I'll play around with it but I think you're right, I think we won't get a github_user anyway.